docs/reST/tut/newbieguide.rst

```diff
@@ -123,93 +127,61 @@ output file had the same pixel format as the input file.  If you're writing a
----------------------
+Be wary of outdated, obsolete, and optional advice.
+---------------------------------------------------
 
+Pygame has been around since the early 2000s, and a lot has changed since then --
+both within the framework itself and within the broader computing landscape as a 
+whole. Make sure to check the dates on materials you read (including this guide!), 
+and take older advice with a grain of salt. Here are some common things that 
+stick out to me:
 
-Hardware surfaces are more trouble than they're worth.
-------------------------------------------------------
+**Dirty Rects**
 
-**Especially in pygame 2, because HWSURFACE now does nothing**
+When you read older bits of pygame documentation or guides online, you may see 
+some emphasis on only updating portions of the screen that are dirty, for the 
+sake of performance (in this context, "dirty" means the region has changed since 
+the previous frame was drawn).  
 
-If you've been looking at the various flags you can use with
-``pygame.display.set_mode()``, you may have thought like this: `Hey,
-HWSURFACE! Well, I want that -- who doesn't like hardware acceleration. Ooo...
-DOUBLEBUF; well, that sounds fast, I guess I want that too!`.  It's not
-your fault; we've been trained by years of 3-d gaming to believe that hardware
-acceleration is good, and software rendering is slow.
+Generally this entails calling :func:`pygame.display.update()` (with a list of 
+rects) instead of :func:`pygame.display.flip()`, not having scrolling backgrounds, 
+or even not filling the screen with a background color every frame because pygame 
+supposedly can't handle it.  Some of pygame's API is designed to support this 
+paradigm as well (e.g. :func:`pygame.sprite.RenderUpdates`), which made a lot of 
+sense in the early years of pygame.
 
-Unfortunately, hardware rendering comes with a long list of drawbacks:
+In the present day though, even modest computers are powerful enough to refresh 
+the entire display once per frame at 60 FPS and beyond.  You can have a moving 
+camera, or dynamic backgrounds and your game should run totally fine.  CPUs are 
+more powerful nowadays, and you can use ``display.flip()`` without fear.
 
- * It only works on some platforms. Windows machines can usually get hardware surfaces if you ask for them. Most other platforms can't. Linux, for example, may be able to provide a hardware surface if X4 is installed, if DGA2 is working properly, and if the moons are aligned correctly. If a hardware surface is unavailable, SDL will silently give you a software surface instead.
+That being said though, there are still plenty of ways to accidentally tank your 
+game's performance with poorly optimized rendering logic.  For example, even on 
+modern hardware it's probably too slow to call ``set_at`` once per pixel on the 
+display surface.  Being mindful of performance is still something you'll have to 
+do.
 
- * It only works fullscreen.
+**HWSURFACE and DOUBLEBUF** 
```

Comment:
Let's not mention these at all?

Reply:
IMHO, this section does do a good job at clearing any potential confusion, many-a-times newbies learning from older tutorials come across these flags, and are confused as to whether these should be used or not (not just newbies, even some more experienced peeps may sometimes misunderstand these flags as well)
So I think this bit should probably stay, IG